docs/functions/data.rst

```diff
@@ -99,6 +99,16 @@ Environment
    .. versionchanged:: 2.0
       Return method changed from stdout.
 
+.. function:: _lp_dirstack() -> var:lp_dirstack
+
+    Returns true if directory stack support is enabled and the directory stack
+    contains more than one directory. In that case, the return variable is set
+    to the number of directories on the stack.
+
+    Can be enabled by :attr:`LP_ENABLE_DIRSTACK`.
```

Comment:
> enabled by

I was going to say this was an error, but since it is disabled by default, I like this actually. I should check the docs for other options that start disabled and can be enabled.